### DISCUSSION TITLE
我的NT有三个媒体库（动漫，电影，电视），转到MP只能改目录结构了吗

### DISCUSSION THREAD

**Discussion by werran2** [jxxghp/MoviePilot#1560] [问题咨询]
下载的目录都在硬盘根目录，媒体库的硬连接目录在分别的link文件夹里。
现在不改目录结构似乎没有什么好的办法可以平滑过渡到MP（直接映射硬盘根目录给MP会不会有什么不可知的危险）

**Comment by dlovew** [comment]
完全没问题，我是/media/download下载目录，里面有movie，comic，tv。与download同级别有/media/hlink，hlink里面也有movie，comic，tv
